model/vehiclelistings.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+class VehicleListings {
+  public $db;
+
+  public function __construct() {
+    $this->db = new mysqli(DB_HOST, DB_USER, DB_PASSWORD, DATABASE);
+  }
+
+  public function getRecordByVin($vin) {
+    $sql = "SELECT * FROM vehicle_listings WHERE vin = '" . $vin . "' LIMIT 1";
+    $result = $this->db->query($sql);
+    $row = $result->fetch_assoc();
+    return $row;
```

Comment:
one option to prevent sql injection is to do a prepare


public function getRecordByVin($vin) {
    $stmt = $this->db->prepare("SELECT * FROM vehicle_listings WHERE vin = ? LIMIT 1");
    $stmt->bind_param("s", $vin); // "s" indicates the type of the parameter, in this case, a string
    $stmt->execute();
    $result = $stmt->get_result();
    $row = $result->fetch_assoc();
    return $row;
}